### ISSUE TITLE
Unable to load the display.html

### ISSUE THREAD

**Issue by eldhomathulla** [airavata-courses/sachinkariyattin#5]
Unable to load the display.html due to the html file being named: Display.html

**Comment by sachinkariyattin** [comment]
@eldhomathulla 
I just fixed the the typo (https://github.com/airavata-courses/sachinkariyattin/commit/afc4185da069d6d93ddbf9c66aeb21ff99edc7bf)

But currently I have pushed RabbitMQ changes to the repo. So, if you clone this particular change you will get all the changes related to the new assignment. So, kindly request you to correct this typo in your local repo and test it.

I will be creating a branch for Assignment 2
